tools/indexer/example/Cargo.toml

```diff
@@ -14,3 +14,4 @@ tracing = "0.1.13"
 tracing-subscriber = "0.2.4"
 
 near-indexer = { path = "../../../chain/indexer" }
+near-primitives = { path = "../../../core/primitives" }
```

Comment:
`near_primitives` are re-exported from `near-indexer`, so instead of adding it as a direct dependency, just `use near_indexer::near_primitives::types::Gas`.

Reply:
Done.